test/format/ops.c

```diff
@@ -1079,6 +1080,8 @@ ops(void *arg)
 		 * a random direction.
 		 */
 		if (positioned) {
+			testutil_assert(F_ISSET(cursor, WT_CURSTD_KEY_INT));
+
```

Comment:
Maybe try moving this assertion into `nextprev` to catch cases where we need `g.dbc_reset`?

`test/format` tries fairly hard to track when it thinks a cursor is positioned -- what I think is happening here is some case where that tracking diverges from reality.